src/playlist.js

```diff
@@ -305,6 +305,39 @@ const calculateExpiredTime = function(playlist, expiredSync, segmentSync) {
 
     return segmentSync.time - sumDurations(playlist, syncIndex, 0);
   }
+  return null;
+};
+
+/**
+ * Calculates the playlist end time or seekable end time depends on using
```

Comment:
I think this description can just be left as `Calculates the playlist end time` since the parameter describes its use